CMakeLists.txt

```diff
@@ -167,6 +167,8 @@ set(XEUS_CLING_SRC
     src/xpaths.cpp
     src/xpaths.hpp
     src/xholder_cling.cpp
+    src/xmagics/executable.cpp
+    src/xmagics/executable.hpp
```

Comment:
(typo) the file is repeated.

Reply:
No, the first is the source file `.cpp` and the second one the header `.hpp`. I made this the same as existing code for other magics.

Reply:
indeed, I was going too quickly over the PR.